frontend/src/scenes/onboarding/error-tracking/OnboardingErrorTrackingSourceMapsStep.tsx

```diff
@@ -1,88 +1,134 @@
-import { useState } from 'react'
+import { useActions, useValues } from 'kea'
 
-import { LemonCollapse } from '@posthog/lemon-ui'
+import { LemonCard } from '@posthog/lemon-ui'
 
 import { CodeSnippet, Language } from 'lib/components/CodeSnippet'
 
 import { OnboardingStepKey } from '~/types'
 
 import { OnboardingStep } from '../OnboardingStep'
+import { SourceMapsInstructionsModal } from './OnboardingErrorTrackingSourceMapsModal'
+import { SourceMapOptionCard } from './source-maps/SourceMapOptionCard'
+import { SourceMapStatus } from './source-maps/SourceMapStatus'
+import { automatedSourceMapsTechnologies } from './source-maps/SourceMapsSDKInstructionsMap'
+import { sourceMapsStepLogic } from './source-maps/sourceMapsStepLogic'
 
 export function OnboardingErrorTrackingSourceMapsStep({ stepKey }: { stepKey: OnboardingStepKey }): JSX.Element {
-    const [activeKey, setActiveKey] = useState<'cli' | 'no-minification' | 'public-source-maps' | undefined>(undefined)
+    const { selectedOption, instructionsModalOpen, shouldShowContinue, shouldShowSourceMapStatus } =
+        useValues(sourceMapsStepLogic)
+    const { setSelectedOption, setInstructionsModalOpen } = useActions(sourceMapsStepLogic)
+
+    const selectedSDK = selectedOption
+        ? automatedSourceMapsTechnologies.find((sdk) => sdk.key === selectedOption)
+        : null
 
     return (
         <OnboardingStep
             title="Link source maps"
             stepKey={stepKey}
-            continueOverride={activeKey !== 'cli' ? <></> : undefined}
-            showSkip={activeKey !== 'cli'}
+            continueOverride={!shouldShowContinue ? <></> : undefined}
+            showSkip={!shouldShowContinue}
         >
             <p>
                 Some languages, bundlers and frameworks obfuscate code making debugging difficult. Source maps are
                 essential to transform code back to the original source. Choose the option that matches your setup to
                 ensure stack traces are readable.
             </p>
-            <LemonCollapse
-                activeKey={activeKey}
-                onChange={(v) => setActiveKey(v ?? undefined)}
-                panels={[
-                    {
-                        key: 'no-minification',
-                        header: 'Unminified code',
-                        content: (
-                            <p>
-                                Not all languages and frameworks are minified. For example Python and Node code often
-                                remains readable and there is no need to upload source maps.
+
+            <div className="space-y-3 mt-4">
+                <SourceMapOptionCard
```

Comment:
Wasn't immediately obvious to me that these were clickable but maybe it's ok... let's see how people get on with it